flask-redis/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python
```

Comment:
Can you specify a tag for the base image please, we trying to advocate best practices and it's easier for us to know if our sample is using a compromised image